cli/src/export.rs

```diff
@@ -1,44 +1,19 @@
-use std::{
-    collections::{BTreeMap, HashMap},
-    ffi::OsString,
-    io::Write,
-    {fs, path::PathBuf},
-};
-
-use clap::{parser::ValueSource, ArgAction, Command};
+use std::{fs, io::Write, path::PathBuf};
 
 use nickel_lang_core::{
-    combine::Combine,
     error::{Error, IOError},
     eval::cache::lazy::CBNCache,
-    identifier::LocIdent,
-    program::{FieldOverride, Program},
-    repl::query_print::{write_query_result, Attributes},
+    program::Program,
     serialize::{self, ExportFormat},
-    term::{
-        record::{Field, RecordData},
-        LabeledType, MergePriority, RuntimeContract, Term, TypeAnnotation,
-    },
-    typ::{RecordRowF, RecordRowsIteratorItem, Type, TypeF},
 };
 
 use crate::{
     cli::GlobalOptions,
     error::{CliResult, ResultErrorExt},
     eval::EvalCommand,
+    input::Prepare,
 };
 
-/// The maximal number of overridable fields displayed. Because there might be a lot of them, we
-/// don't list them all by default.
-const OVERRIDES_LIST_MAX_COUNT: usize = 15;
-
-/// The value name used through the CLI to indicate that an option accepts any Nickel expression as
-/// a value.
-const NICKEL_VALUE_NAME: &str = "NICKEL EXPRESSION";
-
-const EXPERIMENTAL_MSG: &str =
-    "[WARNING] Customize mode is experimental. Its interface is subject to breaking changes.";
-
 #[derive(clap::Parser, Debug)]
 pub struct ExportCommand {
     #[arg(long, value_enum, default_value_t)]
```

Comment:
I guess this can get a short option now, right?

Reply:
Oh right, yes it can.